modules/dmrpp_module/tests_build_dmrpp/Makefile.am

```diff
@@ -20,7 +20,8 @@ AUTOTEST = $(AUTOM4TE) --language=autotest
 TESTSUITE_DMRPP = $(srcdir)/testsuite
 
 check-local: atconfig atlocal  $(TESTSUITE_DMRPP) $(builddir)/test_bes.conf
-	$(SHELL) '$(TESTSUITE_DMRPP)' $(TESTSUITEFLAGS) && echo "bindir: $(bindir)"
+	if ./chkForAWS.sh; then export tsflags="$(TESTSUITEFLAGS) --s3tests=yes"; else export tsflags="$(TESTSUITEFLAGS)"; fi && \
```

Comment:
What happens when the aws command is not installed (or found)?

I think you can decouple these so the '&&' is not required. 

You could simplify the line to 

if ./chkForAWS.sh; then export tsflags="--s3tests=yes"; fi

and then 

$(SHELL) '$(TESTSUITE_DMRPP)' $(TESTSUITEFLAGS) $${tsflags}